automacent-fwk-components/automacent-fwk-core/src/main/java/com/automacent/fwk/core/Driver.java

```diff
@@ -310,18 +310,29 @@ public void startDriver(DriverManagerType driverManagerType) {
 					_logger.info("Using chromeDriver from framework");
 				}
 				ChromeOptions chromeOptions = new ChromeOptions();
-				chromeOptions.addArguments("--no-sandbox");
-				chromeOptions.addArguments("--disable-dev-shm-usage");
-				chromeOptions.addArguments("--safebrowsing-disable-download-protection");
 
-				Map<String, Object> chromePrefs = new HashMap<String, Object>();
-				chromePrefs.put("safebrowsing.enabled", "true");
-				chromeOptions.setExperimentalOption("prefs", chromePrefs);
-
-				_logger.debug("Setting chrome switch --no-sandbox");
-				_logger.debug("Setting chrome switch --disable-dev-shm-usage");
-				_logger.debug("Setting chrome switch --no-sandbox");
-				_logger.debug("Setting chrome pref {safebrowsing.enabled : true}");
+				String debuggerAddress = BaseTest.getTestObject().getDebuggerAddress();
+				if (!debuggerAddress.isEmpty()) {
+					chromeOptions.setExperimentalOption("debuggerAddress", debuggerAddress);
+
+					_logger.debug(String.format("Setting chrome debuggerAddress to %s", debuggerAddress));

```

Comment:
Change message to `Setting chrome experimental option debuggerAddress : %s`

Reply:
changed